README_LINUX.md

```diff
@@ -1,5 +1,5 @@
-Supercollider 3 for linux
-=========================
+Welcome to Supercollider 3.9 for Linux!
```

Comment:
So inviting! ;)

Reply:
yes :)  but should read `SuperCollider`, no?

Reply:
I was blinded by its charm :)